src/components/Overview/PlotlyCharts/ForceDirectedGraph.tsx

```diff
@@ -1,14 +1,174 @@
 import React from 'react';
 import { ForceGraph2D } from 'react-force-graph';
 import myData from '../../../fake-data/force-directed-graph-fake-data.json'; // To be replaced by function transforming AvastarParsedDaat Object in another object with the required shape
+import fakeData from '../../../fake-data/fake-data-agg.json'; // Temporary import while the details information is not automatically fetched from "Who has my data" files
+
 const { useRef } = React;
 
 export default function ForceGraph() {
+  // Read local fake data file
+  const data = fakeData.data_classification;
+
+  // Transform data from State into the right shape for the force graph
+  const setForceGraphData = () => {
+    // Set empty dictionnaries
+    var forceGraphData = [];
+    var nodes = [];
+    var links = [];
+    // Set the initial nodes (You, Facebook, Google)
+    nodes.push({
+      id: 'You',
+      group: 1,
+    });
+    nodes.push({
+      id: 'Facebook',
+      group: 2,
+    });
+    nodes.push({
+      id: 'Google',
+      group: 3,
+    });
+    // Set the initial links (You, Facebook, Google)
+    links.push({
+      source: 'You',
+      target: 'Facebook',
+      value: 10,
```

Comment:
what 10 means ?

Reply:
J'ai ajouté un commentaire dans le code pour ça : `intensity of the moving particles defined relatively by the volume of data transferred`.